<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTS_SalesClass</class>
 <widget class="QMainWindow" name="BTS_SalesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BTS_Sales</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ticket Identifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ticketID">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Student ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="studentID">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>108</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTreeWidget" name="recentTicketsList">
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="separator"/>
   <addaction name="actionCapture"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="bts_sales.qrc">
     <normaloff>:/BTS_Sales/rsrc/key.png</normaloff>:/BTS_Sales/rsrc/key.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bts_sales.qrc">
     <normaloff>:/BTS_Sales/rsrc/lock_unlock.png</normaloff>:/BTS_Sales/rsrc/lock_unlock.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="icon">
    <iconset resource="bts_sales.qrc">
     <normaloff>:/BTS_Sales/rsrc/webcam.png</normaloff>:/BTS_Sales/rsrc/webcam.png</iconset>
   </property>
   <property name="text">
    <string>Start Capture</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop Capture</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bts_sales.qrc"/>
 </resources>
 <connections/>
</ui>
